components/datatypes/productlistitem.schema.json

```diff
@@ -56,6 +56,11 @@
           "title": "Price total",
           "type": "number",
           "description": "The total price for the product line item."
+        },
+        "xdm:unitOfMeasureCode": {
```

Comment:
any reason why we cannot use existing field group https://github.com/adobe/xdm/blob/master/components/fieldgroups/product/product-measurement.schema.json here instead of creating a new property for the same concept